gravitee-apim-repository/gravitee-apim-repository-elasticsearch/src/main/java/io/gravitee/repository/elasticsearch/v4/analytics/AnalyticsElasticsearchRepository.java

```diff
@@ -191,6 +194,17 @@ public Optional<TopFailedAggregate> searchTopFailedApis(QueryContext queryContex
         return this.client.search(indexes, null, esQuery).map(SearchTopFailedApisAdapter::adaptResponse).blockingGet();
     }
 
+    @Override
+    public List<HistogramAggregate<?>> searchHistogram(QueryContext queryContext, HistogramQuery query) {
+        var index = this.indexNameGenerator.getWildcardIndexName(queryContext.placeholder(), Type.V4_METRICS, clusters);
+        var adapter = new SearchHistogramQueryAdapter();
+        var esQuery = adapter.adapt(query);
+        var response = client.search(index, null, esQuery).blockingGet();
+
+        log.debug("Search histogram query: {}", esQuery);
+        return adapter.adaptResponse(response);
```

Comment:
not a blocker but wdyt about writing it like this ?

```suggestion
log.debug("Search histogram query: {}", esQuery);
        return this.client.search(index, null, esQuery).map(SearchHistogramQueryAdapter::adaptResponse).blockingGet();
```

Reply:
I do agree for consistency